include/openthread/backbone_router_ftd.h

```diff
@@ -110,7 +119,11 @@ otBackboneRouterState otBackboneRouterGetState(otInstance *aInstance);
 void otBackboneRouterGetConfig(otInstance *aInstance, otBackboneRouterConfig *aConfig);
 
 /**
- * This function sets the local Backbone Router configuration.
+ * Sets the local Backbone Router configuration #otBackboneRouterConfig. A Server Data Request message
+ * `SRV_DATA.ntf` is initiated automatically if BBR Dataset changes for Primary
+ * Backbone Router.
```

Comment:
We usually have a one line brief and more detailed description in a separate block. Should we maintain this convention?
```suggestion
 * Sets the local Backbone Router configuration #otBackboneRouterConfig.
 *
 * A Server Data Request message `SRV_DATA.ntf` is initiated automatically if BBR Dataset changes for Primary
 * Backbone Router.
```